core/src/main/java/bisq/core/app/BisqExecutable.java

```diff
@@ -100,6 +101,12 @@ protected void doExecute() {
         CoreSetup.setup(config);
         addCapabilities();
 
+        // exit gracefully on shutdown
+        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
+                    gracefulShutDown(() -> log.info("gracefulShutDown complete"));
+                }, "Monitor Shutdown Hook ")
```

Comment:
Is there a reason why you labeled it `Monitor Shutdown Hook `?
```suggestion
                }, "BisqExecutable Shutdown Hook")
```

Reply:
copy and paste... my bad